RENotebookApi/RENotebookApi.fsproj

```diff
@@ -20,6 +27,12 @@
     <None Include="paket.references" />
     <None Include="ReLoad.fsx" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
The packages on most the projects are managed with Paket. We could drop that and use NuGet throughout.

I've found it hard to use solution-wide consistent packages with NuGet as the tooling for that in F# is incomplete.